pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -78,9 +78,17 @@ func GetKubeUtil() (*KubeUtil, error) {
 	return globalKubeUtil, nil
 }
 
-// GetNodeInfo returns the IP address and the hostname of the node where
-// this pod is running.
-func (ku *KubeUtil) GetNodeInfo() (ip, name string, err error) {
+// HostnameProvider kubelet implementation for the hostname provider
+func HostnameProvider(hostName string) (string, error) {
+	ku, err := GetKubeUtil()
+	if err != nil {
+		return "", err
+	}
+	return ku.GetHostname()
```

Comment:
let's be explicit we're returning `ku.GetHostname(), nil`

Reply:
Oh I thought `ku.GetHostname()` already returned the tuple string, error. Do you think it’s worth putting as a comment ?
```go
func (ku *KubeUtil) GetHostname() (string, error)
```
I mirrored the style we used [here](https://github.com/DataDog/datadog-agent/blob/master/pkg/util/docker/global.go#L73).

Reply:
indeed, my bad